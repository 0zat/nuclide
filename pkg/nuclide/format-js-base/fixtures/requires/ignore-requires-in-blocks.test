var c = require('c');
var a = require('a');
var d = require('d');

if (true) {
  var b = require('b');
}

a();
c();
