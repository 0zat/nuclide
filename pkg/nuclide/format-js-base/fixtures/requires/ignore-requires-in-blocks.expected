var a = require('a');
var c = require('c');

if (true) {
  var b = require('b');
}

a();
c();
